make.description.R

```diff
@@ -31,9 +47,9 @@ VignetteBuilder: knitr\n',
 sha <- system('git rev-parse HEAD', intern=TRUE)
 branch <- system('git rev-parse --abbrev-ref HEAD', intern=TRUE)
 
-cat(paste('GithubRepo: spict\n'), file=fn, append=TRUE)
-cat(paste('GithubRef:', branch, '\n'), file=fn, append=TRUE)
-cat(paste('GithubSHA1:', sha, '\n'), file=fn, append=TRUE)
-cat('RoxygenNote: 6.0.1\n', file=fn, append=TRUE)
+#cat(paste('GithubRepo: spict\n'), file=fn, append=TRUE)
+#cat(paste('GithubRef:', branch, '\n'), file=fn, append=TRUE)
+#cat(paste('GithubSHA1:', sha, '\n'), file=fn, append=TRUE)
+#cat('RoxygenNote: 6.0.1\n', file=fn, append=TRUE)
```

Comment:
I removed the github stuff from the DESCRIPTION file. But I am not sure if that is the way to go.

There is another option that could be interesting in an [old stackoverflow question of mine](https://stackoverflow.com/questions/24209336/automating-version-increase-of-r-packages)\

Installing from github adds these lines (and actually the correct ones - this method here is always one step behind - one step is also before -> after a pull request, which means the SHA1 and the repo are not correct)